app/models/like.rb

```diff
@@ -0,0 +1,14 @@
+class Like < ApplicationRecord
+  # belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  # def self.update_likes_counter(post_id)
+  #   post = Post.find(post_id)
+  #   post.update(likes_counter: post.likes.count)
+  # end
+
+  def update_likes_counter(_post_id)
```

Comment:
- [ ] Hi @ahadb802, I don''t understand why you are passing in `_post_id` as an argument in this method